torch/distributed/_tensor/device_mesh.py

```diff
@@ -34,7 +35,10 @@
 class _MeshEnv:
     def __init__(self) -> None:
         self.mesh_stack: List[DeviceMesh] = []
+        # child_to_parent_mapping records all the child to parent mesh mapping in the current Mesh Environment
         self.child_to_parent_mapping: Dict[DeviceMesh, DeviceMesh] = {}
+        # parent_mesh_dim_usage records the PT-D API call on a mesh dimension for a given parent_mesh in the current Mesh Environment
+        self.parent_mesh_dim_usage: Dict[DeviceMesh, Dict[int, str]] = {}
```

Comment:
hmmm early reviewing, but if we want to log it, shouldn't we just add a `log_once` call? why we need to have the usage map?